app/geant-exporter/geant-exporter.cc

```diff
@@ -0,0 +1,246 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file geant-exporter.cc
+//---------------------------------------------------------------------------//
+/*! \brief Geant4 particle definition and physics tables exporter
+ *
+ *  The code exports particle information and physics tables built by the
+ *  selected physics list and geometry. Output data is stored into ROOT files
+ *  and read by Celeritas' Geant Importer class.
+ */
+//---------------------------------------------------------------------------//
+
+// C++
+#include <string>
+#include <vector>
+
+// Project
+#include "src/DetectorConstruction.hh"
+#include "src/ActionInitialization.hh"
+#include "src/GeantPhysicsTableParser.hh"
```

Comment:
I don't mind moving them, but keeping both .hh and .cc files in the same folder follows the same structure found in the rest of Celeritas.

Reply:
I definitely agree the headers and source files should be in the same directory; I just meant that since this is a standalone app all those files can easily live in the same directory as the source file with `main`. Having an extra `src/` directory doesn't buy us much, and in the main repository we also don't have to use the `src/` prefix because of the way the include directories are set up.

Reply:
Oh, ok. I understood your comment as moving only .hh files outside of src/, while keeping the .cc files inside it. I'm fine with removing the src/ folder entirely.